src/api/public/apidocs-new/components/responses/unknown_command_error.yaml

```
@@ -0,0 +1,19 @@
+description: |
+  Bad request.
+
+  XML Schema used for body validation: [status.xsd](../schema/status.xsd)
+content:
+  application/xml; charset=utf-8:
+    schema:
+      $ref: '../schemas/api_response.yaml'
+    examples:
+      unknown_command_error:
+        value:
+          code: unknown_command
+          summary: "Allowed commands are 'register'."
+        summary: Unknown command error
```

```suggestion
      Unknown Command:
        value:
          code: unknown_command
          summary: "Allowed commands are 'register'."
```
Nitpick: I find adding "error" or "_error" kind of redundant, in this context. We can also get rid of the last `summary` tag if we change the name of the example.